pegasus/sites.v3/code.org/views/volunteer_engineer_form.haml

```diff
@@ -59,7 +59,7 @@
   .form-group
     .checkbox
       %label
-        %input#volunteer-after-hoc{name: "volunteer_after_hoc_b", type: "checkbox", value: form.data['volunteer_after_hoc_b'], required: false}/
+        %input#volunteer-after-hoc{name: "volunteer_after_hoc_b", type: "checkbox", value: '1', required: false}/
```

Comment:
Does this give the correct value if the user checks the box, unchecks the box, and submits? What value is sent if the box isn't checked?